src/Wallabag/CoreBundle/Resources/views/themes/material/Entry/entry.html.twig

```diff
@@ -125,7 +125,7 @@
                     {% endif %}
                     {% if craue_setting('share_shaarli') %}
                         <li>
-                        <a href="{{ craue_setting('shaarli_url') }}/index.php?post={{ entry.url|url_encode }}&amp;title={{ entry.title|striptags|url_encode }}&amp;tags={{ entry.tags|join(',')|striptags|url_encode }}" target="_blank" title="shaarli" class="tool icon-image shaarli">
+                        <a href="{{ craue_setting('shaarli_url') }}/index.php?post={{ entry.url|url_encode }}&amp;title={{ entry.title|striptags|url_encode }}&amp;tags={{ entry.tags|join(',')|striptags|url_encode }}&amp;lf_original_url={{ entry.originUrl|url_encode }}" target="_blank" title="shaarli" class="tool icon-image shaarli">
```

Comment:
I can't find documentation about this parameter in Shaarli.

Reply:
This parameter is not handled by Shaarli by default, it's added using my fork of the via plugin: https://github.com/Kdecherf/shaarli-plugin-via

Reply:
Shouldn't we need to write that somewhere? Or do we let user found it on their own?

Reply:
Where can I add a message about it?

Reply:
https://github.com/wallabag/doc/blob/master/en/user/articles/share.md maybe

Reply:
Will add it in a separate PR later. I will probably render this feature deactivatable in another PR.